modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1230,6 +1230,14 @@
      */
     public static final String IGNITE_LOG_CLASSPATH_CONTENT_ON_STARTUP = "IGNITE_LOG_CLASSPATH_CONTENT_ON_STARTUP";
 
+    /**
+     * Index rebuilding parallelism level. If specified, sets the count of threads that are used for index rebuilding
+     * and can only be greater than <code>0</code>, otherwise default value will be used. Maximum count of threads
+     * can't be greater than total available processors count.
+     * Default value is minimum of <code>4</code> and processors count / 4, but always greater than <code>0</code>.
```

Comment:
Why different markup is used for "4"  and for "processors count / 4" ?
Also, it is not clear what are values allowed.

Can it be like: 
"Index rebuilding parallelism level. 
It sets a number of threads will be used for index rebuilding. Zero value means default should be used.
Default value is calculated as "CPU count / 4"

Note: Number of threads is bounded within the range from "1" up to "CPU count".

What do you think?

Reply:
Yes, this version is better, I will fix it.